### PR TITLE
Align rfc1123 header flip, remove artifacts

### PR DESCRIPTION
**Pull request by grundoon**

### CODE REVIEW

File: Tests/LinuxMain.swift [hunk 1]
```diff
@@ -29,6 +29,7 @@ extension HTTPKitTests.HTTPTests {
 		("testUncleanShutdown", testUncleanShutdown),
 		("testClientProxyPlaintext", testClientProxyPlaintext),
 		("testClientProxyTLS", testClientProxyTLS),
+                ("testRFC1123Flip", testRFC1123Flip),
```
Comment (tanner0101) [inline Tests/LinuxMain.swift]: spacing

**Comment by grundoon** [comment]
You mean, why are you requesting `http` and it's returning `https`?  Seems to be on the httpbin.org end – even a client that knows nothing of TLS (e.g. RESTed below) is getting that returned.
```
{
    "args": {},
    "data": "",
    "files": {},
    "form": {},
    "headers": {
        "Accept": "*/*",
        "Accept-Encoding": "gzip, deflate",
        "Accept-Language": "en-us",
        "Host": "httpbin.org",
        "User-Agent": "Rested/2009 CFNetwork/976 Darwin/18.2.0 (x86_64)"
    },
    "json": null,
    "method": "GET",
    "origin": "50.255.0.169, 50.255.0.169",
    "url": "https://httpbin.org/anything"
}
```

**Comment by tanner0101** [comment]
Is this ready to merge?

**Comment by grundoon** [comment]
If you're now seeing correct spacing in LinuxMain, I'd say so.

**Comment by penny-coin** [comment]
Hey @grundoon, you just merged a pull request, have a coin! 

You now have 101 coins.

**Pull request merged**